sklearn/feature_selection/tests/test_base.py

```diff
@@ -1,12 +1,15 @@
 import numpy as np
 from scipy import sparse as sp
+from scipy.stats import pearsonr, spearmanr
```

Comment:
Also, I tried `def call_scorer(X, y=None, **kwargs)` it doesnt work, since SelectKBest.fit() will not allow `y=None` anywhere.